mayavi/tools/helper_functions.py

```diff
@@ -1179,3 +1181,9 @@ def test_triangular_mesh():
     t = np.r_[0, t]
 
     return triangular_mesh(x, y, z, triangles, scalars=t)
+############################################################
+
+def set_picker_props(figure=gcf, pick_type="point_picker",tolerance=0.025):
+    figure().scene.picker.set_picker_props(pick_type, tolerance)
```

Comment:
Why are you passing a callable to get the figure, instead figure should be the result of a `figure()` call.  The default for `figure` could be None and in that case you can call `gcf`.